### PR TITLE
chore: update docs & release history

### PR DESCRIPTION
**Pull request by 0xArdi**
This PR updates the docs and release history for the autonomous-fund.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -62,14 +62,29 @@ In order to run a local demo based on the SMPKit:
...
         ```
+5. Export the necessary environment variables.
 
-5. Build the service deployment.
+    ```bash
+    # set the Goerli RPC endpoint for all the agents in the service
+    export SERVICE_AUTONOMOUS_FUND_RPC_0=YOUR_RPC_ENDPOINT
+    export SERVICE_AUTONOMOUS_FUND_RPC_1=YOUR_RPC_ENDPOINT
+    export SERVICE_AUTONOMOUS_FUND_RPC_2=YOUR_RPC_ENDPOINT
+    export SERVICE_AUTONOMOUS_FUND_RPC_3=YOUR_RPC_ENDPOINT
```
Comment (jmoreira-valory) [inline docs/index.md:72]:
```suggestion
    export SERVICE_AUTONOMOUS_FUND_RPC_0=YOUR_RPC_ENDPOINT_0
    export SERVICE_AUTONOMOUS_FUND_RPC_1=YOUR_RPC_ENDPOINT_1
    export SERVICE_AUTONOMOUS_FUND_RPC_2=YOUR_RPC_ENDPOINT_2
    export SERVICE_AUTONOMOUS_FUND_RPC_3=YOUR_RPC_ENDPOINT_3
```

**Review by jmoreira-valory — APPROVED** [review]

**Review by jmoreira-valory — APPROVED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -62,14 +62,29 @@ In order to run a local demo based on the SMPKit:
...
+    
+   # set all participants
+   export ALL_PARTICIPANTS='[["0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65","0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc","0x976EA74026E726554dB657fA54763abd0C3a0aa9","0x14dC79964da2C08b23698B3D3cc7Ca32193d9955"]]'
```
Comment (Adamantios) [inline docs/index.md:75]: These should be the user's keys. Should we not add something similar to the above?
```suggestion

   export ALL_PARTICIPANTS='[["your_key0","your_key1","your_key2","your_key3"]]'
```

Comment (0xArdi) [reply docs/index.md:75]: they are the keys from above, perhaps we make it clear that they can replace it with those and run it. Or just add a comment that these are the keys from above, and in reality you would need to replace this with your own.

Comment (jmoreira-valory) [reply docs/index.md:75]: These are the addresses in the keys.json file of the service running in the example, aren't they?

Comment (0xArdi) [reply docs/index.md:75]: Yes

File: docs/index.md [hunk 3]
```diff
@@ -62,14 +62,29 @@ In order to run a local demo based on the SMPKit:
...
+   
+   # set the on chain id for the service 
+   export ON_CHAIN_SERVICE_ID=1 
+   ```
+
```
Comment (jmoreira-valory) [inline docs/index.md:80]:
```suggestion
	!!! warning "Important" 
	
		The keys provided in this example are for testing purposes. You must ensure to use your own keys in the `keys.json` file, and ensure that the environment variable `ALL_PARTICIPANTS` matches their addresses.

```

**Review by Adamantios — APPROVED** [review]

**Pull request merged**